timelock-impl/src/main/java/com/palantir/atlasdb/timelock/paxos/AcceptorCacheKey.java

```diff
@@ -27,14 +27,22 @@
 @Value.Immutable
 @JsonDeserialize(as = ImmutableAcceptorCacheKey.class)
 @JsonSerialize(as = ImmutableAcceptorCacheKey.class)
-public interface AcceptorCacheKey {
+public abstract class AcceptorCacheKey {
 
     @JsonValue
     @Value.Parameter
-    UUID value();
+    public abstract UUID value();
 
-    static AcceptorCacheKey newCacheKey() {
+    public static AcceptorCacheKey valueOf(String key) {
+        return ImmutableAcceptorCacheKey.of(UUID.fromString(key));
+    }
+
+    public static AcceptorCacheKey newCacheKey() {
         return ImmutableAcceptorCacheKey.of(UUID.randomUUID());
     }
 
+    @Override
+    public String toString() {
+        return value().toString();
+    }
 }
```

Comment:
Should be fine. This looks like it breaks wire format, but is unavoidable if we want to use this as a map key, and shouldn't be used anywhere yet